packages/experimental/Checkbox/MIGRATION.md

```diff
@@ -0,0 +1,106 @@
+# Checkbox Migration
+
+## STATUS: WIP 🚧
+
+This Migration guide is a work in progress and is not yet ready for use.
+
+## Migration from v0 Checkbox
+
+### Component renames
+
+| v0 `Checkbox`                       | v1 `Checkbox`                       |
+| ----------------------------------- | ----------------------------------- |
+| `<Checkbox label="Hello, World" />` | `<Checkbox label="Hello, World" />` |
+
+### Props that remain as is
+
+- Any props that are part of `ViewProps`
+- `checked`
+- `componentRef`
+- `defaultChecked`
+- `label`
+- `onChange`
+- `tooltip`
+
+### Props no longer supported with an equivalent functionality in v1 Checkbox
```

Comment:
Nit: This statement confused me a bit, run-on sentence?

Maybe "Props that have been re-purposed" or "Updated Props"